drape/drape_tests/glyph_mng_tests.cpp

```diff
@@ -11,16 +11,25 @@
 
 #include "platform/platform.hpp"
 
+#include "std/target_os.hpp"
+
 #include <QtGui/QPainter>
 
-#include <ft2build.h>
 #include <functional>
 #include <iostream>
 #include <vector>
+
+#include <ft2build.h>
 #include FT_FREETYPE_H
 #include FT_MODULE_H
-#include <harfbuzz/hb-ft.h>
+// A system dependency is used on these platforms. TODO(AB): There should be a way to build everything from sources.
+#if defined(OMIM_OS_LINUX) || defined(OMIM_OS_WINDOWS)
 #include <harfbuzz/hb.h>
+#include <harfbuzz/hb-ft.h>
+#else
+#include <hb.h>
+#include <hb-ft.h>
+#endif
```

Comment:
```suggestion
#include <hb.h>
#include <hb-ft.h>
```
`.../include/harfbuzz` is in `harfbuzz::harfbuzz`'s `INTERFACE_INCLUDE_DIRECTORIES`.

Reply:
@Osyotr can you please clarify your comment? Did you test it on Linux?

Reply:
`harfbuzz-config.cmake` from ubuntu 24.04:
```cmake
set_target_properties(harfbuzz::harfbuzz PROPERTIES
  INTERFACE_INCLUDE_DIRECTORIES "${PACKAGE_PREFIX_DIR}/include/harfbuzz"
  ...
)
```

So if `#include <hb.h>` doesn't work for you then you're not linking against `harfbuzz::harfbuzz`: https://github.com/organicmaps/organicmaps/blob/088bef549e4e731304f2c48681141f1485b5cb84/drape/CMakeLists.txt#L177

Reply:
@Osyotr Now it still doesn't work.

Reply:
Ubuntu 22 has this:
```
$ cat /usr/lib/x86_64-linux-gnu/cmake/harfbuzz/harfbuzz-config.cmake | grep INTERFACE_INCLUDE_DIRECTORIES
  INTERFACE_INCLUDE_DIRECTORIES "${_harfbuzz_prefix}/${_harfbuzz_includedir}/harfbuzz"
  INTERFACE_INCLUDE_DIRECTORIES "${_harfbuzz_prefix}/${_harfbuzz_includedir}/harfbuzz"
  INTERFACE_INCLUDE_DIRECTORIES "${_harfbuzz_prefix}/${_harfbuzz_includedir}/harfbuzz"
    INTERFACE_INCLUDE_DIRECTORIES "${_harfbuzz_prefix}/${_harfbuzz_includedir}/harfbuzz"
```

Reply:
```
$ ls -a /usr/include/harfbuzz/
.                hb-blob.h        hb-draw.h  hb-glib.h             hb-graphite2.h  hb-ot-deprecated.h  hb-ot-meta.h     hb-ot-var.h      hb-shape.h    hb.h
..               hb-buffer.h      hb-face.h  hb-gobject-enums.h    hb-icu.h        hb-ot-font.h        hb-ot-metrics.h  hb-ot.h          hb-style.h
hb-aat-layout.h  hb-common.h      hb-font.h  hb-gobject-structs.h  hb-map.h        hb-ot-layout.h      hb-ot-name.h     hb-set.h         hb-unicode.h
hb-aat.h         hb-deprecated.h  hb-ft.h    hb-gobject.h          hb-ot-color.h   hb-ot-math.h        hb-ot-shape.h    hb-shape-plan.h  hb-version.h
```

Reply:
Sorry, my fault, accidentally removed `::harfbuzz` suffix. Thanks for the help!